.github/workflows/test-microk8s.yaml

```diff
@@ -1,69 +1,54 @@
-name: MicroK8s Test
```

Comment:
Completely agreed.  I didn't mess with it just because this is what was here, but I made a mental note of "we should get consistent".  I'll pull the same name as we have on the other charms